profile/CSIP.xml

```diff
@@ -187,7 +187,7 @@
             <requirement ID="CSIP4" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentInformationTypeSpecification" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                <description>
                     <head>Specific content type</head>
-                   <p xmlns="http://www.w3.org/1999/xhtml">An added attribute which describes the specific content information type specification used for the transferred content. The attribute is mandatory to use when the METS document describes a representation. The vocabulary is going to evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
+                   <p xmlns="http://www.w3.org/1999/xhtml">Used to declare the specific content information type specification used for the transferred content. The attribute is mandatory for representation level METS documents. The vocabulary will evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
```

Comment:
```suggestion
                    <p xmlns="http://www.w3.org/1999/xhtml">Used to declare the Content Information Type Specification that was used in creating the package. Legal values are defined in a fixed vocabulary. The attribute is mandatory for representation level METS documents. The vocabulary will evolve under the care of the DILCIS Board as additional Content Information Type Specifications are developed.</p>
```

>mandatory for representation level METS documents

There are only a few CIT specifications, so many cases will not have a suitable CITS: a private person's documents (i.e. not managed with an ERMS), a photo collection, a film (incl. the script, storyboard, costume sketches, audio, video, trailers, posters etc.). So the attribute is mandatory, but CITS itself is not.

I propose adding the value "NONE" to the vocabulary. The terms refer to specific CITS, so "OTHER" is not a logically correct value for none. "OTHER" would go for an in-house CITS, e.g. if a film archive develops one for internal use with films.

"NONE" would not trigger the use of `@csip:OTHERCONTENTINFORMATIONTYPE`, only "OTHER" would do it.